src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -93,6 +93,35 @@
  *  Inet4Address.ofLiteral("02130706689"); // ==> /127.0.1.1
  * }
  *
+ * <p> The above forms adhere "strict" decimal-only syntax.
+ * Additionally, the following (loose) syntax is available with
+ * {@link Inet4Address#ofPosixLiteral(String)} method that implements
+ * <a href="https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_addr.html">
+ * POSIX {@code inet_addr}</a> compatible parsing algorithm, allowing
+ * octal and hexadecimal address segments. Please refer to
+ * <a href="https://www.ietf.org/rfc/rfc6943.html#section-3.1.1"> <i>RFC&nbsp;
+ * 6943: Issues in Identifier Comparison for Security Purposes</i></a>.
+ * <p> The following (non-decimal) forms are supported
+ * in this mode:
+ * {@snippet :
+ *  // Dotted-quad 'x.x.x.x' form with four part address literal
+ *  Inet4Address.ofPosixLiteral("0177.0000.0000.0001"); // ==> /127.0.0.1
+ *  Inet4Address.ofPosixLiteral("0127.0.0.1"); // ==> /87.0.0.1
+ *
+ *  // Dotted-triple 'x.x.x' form with three part address literal,
+ *  // the last part is placed in the rightmost two bytes
+ *  // of the constructed address
+ *  Inet4Address.ofPosixLiteral("0x7F.0.0x101"); // ==> /127.0.1.1
+ *
+ *  // Dotted-double 'x.x' form with two part address literal,
+ *  // the last part is placed in the rightmost three bytes
+ *  // of the constructed address
+ *  Inet4Address.ofPosixLiteral("0177.0200401"); // ==> /127.1.1.1
+ *
+ *  // Dotless 'x' form with one value that is stored directly in
+ *  // the constructed address bytes without any rearrangement
+ *  Inet4Address.ofPosixLiteral("017700000001"); // ==> /127.0.0.1
+ * }
```

Comment:
Please move this part into the normative part of the `ofPosixLiteral` method description